### ISSUE TITLE
Each Loading Pepe should give unobtrusive loading indicator (spinner)

### ISSUE THREAD

**Issue by crmn111** [CryptoPepes/cryptopepes-alpha-test#34]
Each Loading Pepe should give unobtrusive loading indicator (spinner)

Currently each pepe has a grey background. Each Loading Pepe should give unobtrusive loading indicator (spinner).

### Affected parts

- [ ] Backend API: search/lookups
- [ x] Frontend: Visual
- [ ] Frontend: interactions
- [ ] Smartcontract: CPEP erc271
- [ ] Smartcontract: Cozy auction
- [ ] Smartcontract: Sale auction
- [ ] Smartcontracts - miscellaneous
- [ ] Miscellaneous

<img width="851" alt="screen shot 2018-06-19 at 13 49 01" src="https://user-images.githubusercontent.com/881181/41595621-d36e5b06-73c7-11e8-92c0-4b2044a68e4b.png">

**Comment by kitblake** [comment]
We need some sorta cache for the SVGs somewhere along the traject. 
Is the server currently generating each Pepe on demand?

**Comment by programmerror** [comment]
@kitblake No, it's pulling an already minimized SVG from a datastore. And images are cached per user now, for 20 mins. This short amount of time was chosen because of pepes being subject to change shortly after being born (orphan blocks). Maybe increasing it to 1 year would be good, but conditionally; only if there are enough confirmations.

**Comment by kitblake** [comment]
Ok. Maybe don't cache newborns?
 
"there are only two hard problems in software development; they are: cache invalidation, naming things, and off-by-one errors." 
– Toby Speight Dec 14 '15 at 14:44

**Comment by kitblake** [comment]
@dirkjan111 Instead of that gray background with a spinner, it would look better if a default Pepe background image was displayed, which would then be replaced/covered by the SVG once it arrives.

**Comment by crmn111** [comment]
A loading indicator is a very common way to communicate a waiting time .... but it shouldn't be obtrusive, doesn't have to be animated even, we have to experiment with the layout. Using a default Pepe could work as well, faded out to prevent graphical overload .... perhaps in combination with a spinner.

**Comment by kitblake** [comment]
Yeah, maybe a faded Pepe. That would look less empty than the gray.
